pypeit/core/flexure.py

```diff
@@ -303,147 +305,226 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, arx_lines, mxshft=20):
                 corr_cen=corr.size/2, smooth=smooth_sig_pix, success=success)
 
 
```

Comment:
Can we change this bpm to slit_bpm or something like that. Usually bpm in PypeIt is reserver for the actual bpm image.